pkg/rebuild/schema/schema.go

```diff
@@ -98,6 +102,20 @@ type Message interface {
 	Validate() error
 }
 
+// TODO: Most callers of this can be replace with a stub instead.
```

Comment:
Right now it's used by both tools/benchmark/run.go and tools/ctl.go so it can't be private (yet).

In tools/benchmark/run.go we can't use stubs easily because both the executors are creating these http request objects as part of the interface.

 I think the tools/ctl.go code is just run-one, which can re-use the tools/benchmark/run.go code. I'll change that in a followup PR which can also move this method to be private inside the benchmark package. After that, it's not as important to rething the stub API I think.

Reply:
This is the commit on a different branch. I'll rebase and open that as a PR once this one is submitted.

https://github.com/google/oss-rebuild/commit/6d17b38e7a4b17311f81b0cba088de6da4a421f1

Reply:
okay this is a super nit I grant you but an alternative would be to inline the function in both places and then migrate to stubs and remove one of them. the super nit logic to that would be this file's git log wouldn't be impacted by the adding-then-removal. persnickity so you do you.

Reply:
I'll leave it as is, easier to just merge and the followup with the next PR.